content/tokio/topics/shutdown.md

```diff
@@ -69,33 +69,63 @@ async fn main() {
 
 ## Telling things to shut down
 
-The most common tool for telling every part of the application to shut down is
-to use a [broadcast channel][broadcast]. The idea is simple: every task in the
-application has a receiver for the broadcast channel, and when a message is
-broadcast on the channel, the tasks shut themselves down. Usually, receiving
-this broadcast message is implemented using [`tokio::select`][select]. For
-example, in mini-redis each task receives shutdown in this way:
+When you want to tell one or more tasks to shut down, you can use [Cancellation 
+Tokens][cancellation-tokens]. These tokens allow you to notify tasks that they 
+should terminate themselves in response to a cancellation request, making it 
+easy to implement graceful shutdowns.
+
+Note that, a `CancellationToken` can only be used by one task at a time due to 
+ownership rule. To use the same `CancellationToken` in multiple tasks you can make 
```

Comment:
The grammar here is a bit awkward. How about this?

> To share a cancellation token between several tasks, you must clone it. This is due to the single ownership rule that requires that each value has a single owner. When cloning a token, you get another value that's indistinguishable from the original; if one is cancelled, then the other is also cancelled. You can make as many clones as you need, and when you call `cancel` on one of them, they're all cancelled.

Reply:
Yeah, this is a good explanation. I will add this.

Reply:
@Darksonn I have updated the content